src/components/chart/_macro-options.md

```diff
@@ -1,27 +1,28 @@
 |                          |
-| annotations              | array   | false    | An array of point annotations.                                                                                                                                                               |
-| rangeAnnotations         | array   | false    | An array of range annotations.                                                                                                                                                               |
-| referenceLineAnnotations | array   | false    | An array of reference line annotations.                                                                                                                                                      |
-| estimateLineLabel        | string  | false    | Label for the estimate line in the legend. Used in column charts with confidence intervals.                                                                                                  |
-| uncertaintyRangeLabel    | string  | false    | Label for the confidence interval in the legend. Used in column charts showing uncertainty ranges.                                                                                           |
+| annotations              | array   | false                  | An array of point annotations. Will be ignored if an iframe url is set.                                                                                                                                                    |
+| rangeAnnotations         | array   | false                  | An array of range annotations. Will be ignored if an iframe url is set.                                                                                                                                                    |
+| referenceLineAnnotations | array   | false                  | An array of reference line annotations. Will be ignored if an iframe url is set.                                                                                                                                           |
+| estimateLineLabel        | string  | false                  | Label for the estimate line in the legend. Used in column charts with confidence intervals. Will be ignored if an iframe url is set.                                                                                       |
+| uncertaintyRangeLabel    | string  | false                  | Label for the confidence interval in the legend. Used in column charts showing uncertainty ranges. Will be ignored if an iframe url is set.                                                                                |
+| iframeURL                | string  | false                  | A url for a visualisation to display in an iframe instead of displaying a Highcharts chart. Will be ignored if an iframe url is set.                                                                                       |
```

Comment:
```suggestion
| iframeURL                | string  | false                  | A url for a visualisation to display in an iframe instead of displaying a Highcharts chart.                                                                                   |
```

Reply:
We dont need this statement here

Reply:
Oops - now removed.